polyfill.js

```diff
@@ -1,9 +1,5 @@
 if (!('fromEntries' in Object)) {
   Object.fromEntries = iter => {
-    if (iter == null) {
-      return {};
-    }
-
     const pairs = [ ...iter ]; // calls GetIterator (Array.from would coerce)
```

Comment:
fwiw i think `Array.from` would work fine here; `[...iter]` produces an array too, but Array.from would provide the ability to use a mapper function and avoid the for loop.

Separately, this `pairs` thing isn't necessary at all with the loop, since you can just `for..of` over `iter`.

Reply:
Array.from would permit array-like input. Currently that’s not the case — do you think it should? Map and Set do not, but I believe all the TypedArray.from methods and Array.from do.

Reply:
(Gonna merge this and open a follow-up issue regarding that)

Reply:
ahhh good call. no, it probably shouldn't.